docs/endpoints/campaigns.md

```diff
@@ -0,0 +1,75 @@
+## Campaigns
+
+## Retrieve All Campaigns (created in Rogue)
+
+```
+GET /api/v3/campaigns
+```
+
+Example Response:
+
+```
+{
+    "data": [
+        {
+            "id": 9000,
+            "internal_title": "Test campaign",
+            "start_date": "2018-10-10 00:00:00",
+            "end_date": "2018-12-10 00:00:00",
+            "created_at": {
+                "date": "2018-10-24 19:02:06.000000",
+                "timezone_type": 3,
+                "timezone": "UTC"
+            },
```

Comment:
The date formats in this example are inconsistent. We started pullling this data over in https://github.com/DoSomething/gambit-content/pull/1105 --  I've noticed that the current `end_date` property is returned as an object, not a string. We've previously gotten burned on the Gambit side from an [unannounced format change to end date](https://github.com/DoSomething/gambit-content/pull/1097/files).

Does it make sense to return these dates as strings instead of objects, to keep consistent with formatting of date properties in the other endpoints?

Reply:
PR [here](https://github.com/DoSomething/rogue/pull/803)!